subsys/bootloader/Kconfig

```diff
@@ -90,7 +90,7 @@ config SB_SIGNING_PUBLIC_KEY
 	  Path to a PEM file.
 	  When using a custom signing command, specify the corresponding public
 	  key here. This public key is checked during building, and added to
-	  the provision page as the first entry. See SB_PUBLIC_KEY_FILES.
+	  the provisioning data as the first entry. See SB_PUBLIC_KEY_FILES.
```

Comment:
added to the the provisioning data => Added as the first entry to be provisioned